dashboard/lib/resources_autocomplete.rb

```diff
@@ -0,0 +1,12 @@
+class ResourcesAutocomplete < AutocompleteHelper
+  def self.get_search_matches(query, limit)
+    limit = format_limit(limit)
+
+    rows = Resource.limit(limit)
+    query = format_query(query)
+    return [] if query.length < MIN_WORD_LENGTH + 2
```

Comment:
Why + 2?

Reply:
The query adds a symbol on each side of the word. I'll add a comment.

Reply:
Actually, I'm going to move it up a line and not bother with the +2 thing